src/components/book.js

```diff
@@ -2,27 +2,63 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/booksSlice';
+import ProgressBar from './progress';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   return (
-    <>
+    <div className="Lesson-Panel">
       <p key={book.id}>
-        <strong>{book.title}</strong>
+        <strong className="Title">{book.title}</strong>
+        <br />
         {' '}
-        <span>by</span>
         {' '}
-        <strong>{book.author}</strong>
+        <strong className="author">{book.author}</strong>
       </p>
-      <button
-        type="button"
-        onClick={() => {
-          dispatch(removeBook(book.id));
-        }}
-      >
-        Delete
-      </button>
-    </>
+      <ul className="btn-container">
+        <li>
+          <button className="Comments Remove" type="button">
+            Comments
+          </button>
+        </li>
+        <div className="Line-2" />
+        <li>
+          <button
+            type="button"
+            onClick={() => {
+              dispatch(removeBook(book.id));
+            }}
+            className="Remove"
+          >
+            Remove
+          </button>
+        </li>
+        <div className="Line-2" />
+        <li>
+          <button className="Edit Remove" type="button">
+            Edit
+          </button>
+        </li>
+      </ul>
+      <div className="book-progress">
+        <ProgressBar />
+        <div className="progress-text">
+          <p className="Percent-Complete">
+            {Math.floor(Math.random() * (99 - 0)) + 0}
+            <span>%</span>
+          </p>
+          <p className="Completed">Completed</p>
+        </div>
+        <hr className="vertical-line" />
+        <div className="chapters">
+          <p className="Current-Chapter">CURRENT CHAPTER</p>
+          <p className="Current-Lesson">Chapter 12</p>
+          <button type="button" className="Update-progress">
+            UPDATE PROGRESS
+          </button>
+        </div>
+      </div>
```

Comment:
- [ ] @Leeoasis great work implementing the changes requested, however, the thickness of the progressive bar is different from the zeplin design given, check the comparison below:


Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/230710093-233f0e77-7412-4ef4-8c73-340a022d8ea5.png)  | ![image](https://user-images.githubusercontent.com/61408860/230710138-a6f86b59-d66e-482a-b532-16f4a40f71e6.png)